app/src/main/res/navigation/main_nav.xml

```diff
@@ -60,11 +58,15 @@
 
         <action
             android:id="@+id/action_to_add_safe_nav"
-            app:destination="@id/add_safe_nav"
+            app:destination="@id/navigation"
             app:launchSingleTop="true"
             app:popUpTo="@id/safeBalancesFragment"
             app:popUpToInclusive="false"/>
 
     </fragment>
+    <navigation android:id="@+id/navigation"
+        app:startDestination="@id/add_safe_nav">
+        <include app:graph="@navigation/add_safe_nav" />
+    </navigation>
```

Comment:
Why is this additional level of nesting necessary?

Reply:
I don't remember touching this file.

Reply:
It seems this was merged in from master.